content/realtime/channels/overview.textile

```diff
@@ -140,9 +140,29 @@ bc[objc]. ARTRealtimeChannel *channel = [realtime.channels get:@"channelName"];
 
 bc[swift]. let channel = realtime.channels.get("channelName")
 
-h4(#setting-channel). Setting channel options and encryption
+h4(#setting-channel-options). Setting channel options
 
-A set of "channel options":#channel-options may also be passed to configure a channel for encryption. Find out more about "symmetric message encryption":/realtime/encryption.
+You can set "channel options":#channel-options in one of two ways:
+
+* Initially, when you first obtain a channel using "channels.get":#obtaining-channel.
+* After creation, when you update a channel using "channels.setOptions":#modifying-options.
+
+h5(#setting-channel-parameters). Setting channel parameters
+
+Channel parameters allow a client to set the properties of a channel. The are two parameters that can be specified:
+
+* "rewind":/realtime/channels/channel-parameters/rewind
+* "deltas":/realtime/channels/channel-parameters/deltas
+
+bc[jsall]. const realtime = new Ably.Realtime('{{API_KEY}}');
```

Comment:
@QuintinWillison could we get a version of this code sample that uses deltas as well as rewind?

Reply:
```js
const channelOptions = {
  params: {
    delta: 'vcdiff',
  },
};
```

And, for what it's worth, my preference is to _never_ use abbreviations in variable names - especially not in example code where they can easily be misinterpreted. Clarity is key. Thus, `Options` is always preferable to `Opts`.

Reply:
(ignoring "params", of course 😬)

Reply:
Thanks, I totally agree, I think it's something we as a company need to create a code style guide around. There are many more examples of this in the docs and potential even the tutorials